src/cli-constant.js

```diff
@@ -1,64 +1,244 @@
 "use strict";
 
```

Comment:
Would be nice to have a comment here explaining what each of the properties on an option do.

Reply:
Done.